pkg/cmd/run.go

```diff
@@ -688,7 +694,11 @@ func init() {
 	runCmd.Flags().StringP("config", "c", "", "config (e.g. dev)")
 	runCmd.RegisterFlagCompletionFunc("config", configNamesValidArgs)
 	runCmd.Flags().String("command", "", "command to execute (e.g. \"echo hi\")")
-	runCmd.Flags().Bool("preserve-env", false, "ignore any Doppler secrets that are already defined in the environment. this has potential security implications, use at your own risk.")
+	// note: requires using "--preserve-env=VALUE", doesn't work with "--preserve-env VALUE"
```

Comment:
Is this a Cobra idiosyncrasy?

Reply:
It is, afaik. Their [docs](https://github.com/spf13/pflag#command-line-flag-syntax) indicate that a space is only allowed on "non-boolean and flags without a 'no option default value'", and this flag now has a "no option default value".

Reply:
Well, that's annoying. Will this be reflected in the help output automatically? Or should we update the flag description?

Reply:
We're still waiting on reviews from the rest of the team but should have this shipped this week.

> Well, that's annoying. Will this be reflected in the help output automatically? Or should we update the flag description?

I'll add a note to the flag description.